### DISCUSSION TITLE
Spark SQL via dbt-athena

### DISCUSSION THREAD

**Discussion by u-ra-ra-ra** [dbt-athena/dbt-athena#580] [General]
Hey Community,

AWS athena recently allowed running Spark SQL.
Is there any guidance on supporting this via dbt-athena-commnuity? 

My use case:
I am trying to run certain workload via Spark SQL and others via Athena interactive query.

Thanks in advance for any information!

**Comment by nicor88** [comment]
We don't support directly SparkSQL in the dbt models, but since https://github.com/dbt-athena/dbt-athena/pull/248 was released, in 1.7.2 you can should be able to use  SparkSQL via python models submissions.

Using SparkSQL natively in dbt-athena, instead of trino sql, it will require a bigger refactor.
